rust-runtime/aws-smithy-runtime-api/src/client/connection.rs

```diff
@@ -25,6 +26,10 @@ impl ConnectionMetadata {
     }
 
     /// Create a new [`ConnectionMetadata`].
+    #[deprecated(
+        since = "1.0.3",
```

Comment:
The next release will be a minor, so this should be:
```suggestion
        since = "1.1.0",
```